include/image.mk

```diff
@@ -589,12 +604,16 @@ endef
 
 define Device/DumpInfo
 Target-Profile: DEVICE_$(1)
-Target-Profile-Name: $(DEVICE_TITLE)
+Target-Profile-Name: $(DEVICE_TITLE)$(if $(strip $(DEVICE_ALT0_TITLE)), ($(DEVICE_ALT0_TITLE)))
 Target-Profile-Packages: $(DEVICE_PACKAGES)
 Target-Profile-hasImageMetadata: $(if $(foreach image,$(IMAGES),$(findstring append-metadata,$(IMAGE/$(image)))),1,0)
 Target-Profile-SupportedDevices: $(SUPPORTED_DEVICES)
 Target-Profile-Description:
 $(DEVICE_DESCRIPTION)
+$(if $(strip $(DEVICE_ALT0_TITLE)),Alternative device titels:)
+$(DEVICE_ALT0_TITLE)
```

Comment:
Indentation? The output would be more (machine-) readable having the alternative titles indented by two spaces or a tab.

Reply:
The machine readable part is where the json comes in game, I don't see any machine parse the description, or?

Reply:
But indenting by two spaces wouldn't hurt, would it? It might actually look nicer, I will try later ...

Reply:
Indenting don't work, neither with spaces nor tabs, however I added a single dash in front which should do the job

Reply:
Looks good, and works for different number of ALT# (just tested).